apps/meteor/app/livechat/imports/server/rest/users.ts

```diff
@@ -39,14 +40,14 @@ API.v1.addRoute(
 				}
 
 				return API.v1.success(
-					await findAgents({
+					(await findAgents({
 						text,
 						pagination: {
 							offset,
 							count,
 							sort,
 						},
-					}),
+					})) as { users: ILivechatAgent[]; count: number; offset: number; total: number },
```

Comment:
how difficult would it be to convert `findAgents` and `findManagers` to TS? :awesome-face: